examples/host/cdc_msc_hid/src/cdc_app.c

```diff
@@ -89,7 +89,7 @@ void tuh_cdc_mount_cb(uint8_t idx) {
   // while eneumerating new cdc device
   cdc_line_coding_t line_coding = {0};
   if (tuh_cdc_get_local_line_coding(idx, &line_coding)) {
-    printf("  Baudrate: %lu, Stop Bits : %u\r\n", line_coding.bit_rate, line_coding.stop_bits);
+    printf("  Baudrate: %" PRIu32 ", Stop Bits : %u\r\n", line_coding.bit_rate, line_coding.stop_bits);
```

Comment:
gcc print format for uint32_t is %lu while clang is %u, should use `PRIu32` to prevent format warning